### ISSUE TITLE
Derive from a different base image in Dockerfile

### ISSUE THREAD

**Issue by hferentschik** [jenkins-x/ext-jacoco#60]
The idea is to avoid

```
COPY ./tmp/ca-certificates.crt /etc/ssl/certs/
```
and the coressponding code in the Makefile and its _cert_ target.

The reason this exists atm is that the _scratch_ image does not contain certificates.

**Comment by hferentschik** [comment]
Ideally, I would like to use a multi-stage build, but the current Docker client version in use does not allow this. 

```
FROM alpine:latest as certs
RUN apk --update add ca-certificates

FROM scratch
ENV PATH=/bin

COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
COPY ./build/jx-app-jacoco /jx-app-jacoco

EXPOSE 8080
ENTRYPOINT ["/jx-app-jacoco"]
```

See also https://github.com/jenkins-x-apps/jx-app-jacoco/pull/67#issuecomment-460554889

**Comment by hferentschik** [comment]
Merged via pull request #67
